### PR TITLE
Adding r-bwstools

### PR DESCRIPTION
**Pull request by aspencage**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by aspencage** [comment]
@conda-forge/help-r @mfansler I believe this should now be ready. The linux and OSx builds are succeeding, the win build is not, but this should not be an issue.

File: recipes/r-bwstools/MIT.txt
Comment (mfansler) [inline recipes/r-bwstools/MIT.txt:1]: This file should be removed. Instead, we package the common MIT license text delivered by the `r-base` package, i.e., that's what the following does:

```yaml
about:
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/MIT
```

File: recipes/r-bwstools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    and to analyze data from these designs, using aggregate and individual methods such
+    as: difference scores, Louviere, Lings, Islam, Gudergan, & Flynn (2013) <doi:10.1016/j.ijresmar.2012.10.002>;
+    analytical estimation, Lipovetsky & Conklin (2014) <doi:10.1016/j.jocm.2014.02.001>;
+    empirical Bayes, Lipovetsky & Conklin (2015) <doi:10.1142/S1793536915500028>; Elo,
+    Hollis (2018) <doi:10.3758/s13428-017-0898-2>; and network-based measures.'
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+# Type: Package
```
Comment (mfansler) [inline recipes/r-bwstools/meta.yaml:62]: Drop uninformative comment.

```suggestion
```

File: recipes/r-bwstools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - r-rlang
+    - r-tidyr
+
+test:
+  commands:
+    - $R -e "library('bwsTools')"           # [not win]
+    - "\"%R%\" -e \"library('bwsTools')\""  # [win]
+
+about:
+  home: https://github.com/markhwhiteii/bwsTools
```
Comment (mfansler) [inline recipes/r-bwstools/meta.yaml:50]: Let's also include the OSF site they reference from GitHub:

```suggestion
  home: https://osf.io/xftvq/
  dev_url: https://github.com/markhwhiteii/bwsTools
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this! ❤️ 

Mostly good, but I have noted a few things to tidy up before merging.

**Comment by aspencage** [comment]
> Thanks for submitting this! ❤️
> 
> Mostly good, but I have noted a few things to tidy up before merging.

Thanks @mfansler! I have committed your suggestions and removed the extraneous license files

**Review by mfansler — APPROVED** [review]

**Pull request merged**